common/scripted_effects/wc_race_effects.txt

```diff
@@ -177,6 +177,16 @@ set_race_effect = {
 		limit = { scope:temp_race = flag:creature_kvaldir }
 		become_undead_effect = yes
 	}
+
+	if = { # Starts mana regeneration loop
```

Comment:
I though you made it on the trait/perk adding and that's why you need my review.

Reply:
As far as I know, it's impossible to add it on perk receiving.

Reply:
I tried